packages/slate/src/models/node.js

```diff
@@ -1351,6 +1351,41 @@ class Node {
     return path
   }
 
+  /**
+   * refind the path of node if path is changed
+   *
+   * @param {Array} path
+   * @param {Node|String} node
+   * @return {Array}
+   */
+
+  refindPath(path, node) {
```

Comment:
These methods should take `key` instead of `node`, to match how we handle it elsewhere I think.